pkg/bucketeer/model/metrics_event.go

```diff
@@ -31,7 +31,12 @@ type metricsDetailEventType string
 
 type APIID int
 
+// The API IDs must match the IDs defined on the main repository
+// https://github.com/bucketeer-io/bucketeer/blob/main/proto/event/client/event.proto
 const (
-	GetEvaluation  APIID = 1
-	RegisterEvents APIID = 3
+	GetEvaluation   APIID = 1
+	RegisterEvents  APIID = 3
+	GetFeatureFlags APIID = 4
+	GetSegmentUsers APIID = 5
+	SDKGetVariation APIID = 100
```

Comment:
New API IDs.
The `SDKGetVariation` will be used to report metrics related to local evaluations.

Reply:
Now that we have imported the main Bucketeer repo, I think we can directly use the API ID from it. 
Therefore, we don't need to worry about keeping this place in sync with the main repo. What do you think about this?

Reply:
The ID must be equal to. Otherwise, the envoy will fail while converting it from JSON to gRPC format.
If we choose to use it directly, we should do so in all other places, not just here.
We could do this when moving to the gRPC protocol because we might use gRPC for server SDKs in the future.
WDYT?

Reply:
Yeah, it should be fine for the REST API. Let's keep this! 👍